README.md

```diff
@@ -3,38 +3,55 @@
 [![GoReportCard](http://goreportcard.com/badge/elastic/beats)](http://goreportcard.com/report/elastic/beats)
 [![codecov.io](https://codecov.io/github/elastic/beats/coverage.svg?branch=master)](https://codecov.io/github/elastic/beats?branch=master)
 
-# Beats - Lightweight shippers for Elasticsearch & Logstash
+# Beats - The Lightweight Shippers of the Elastic Stack
 
-The [Beats](https://www.elastic.co/products/beats) are lightweight processes,
-written in Go, that you install on your servers to capture all sorts of
-operational data like logs, operating system metrics or network packet data,
-and to send it to Elasticsearch, either directly or via Logstash, so it can be
-visualized with Kibana.
+The [Beats](https://www.elastic.co/products/beats) are lightweight data
+shippers, written in Go and having no runtime dependencies, that you install on your
+servers to capture all sorts of operational data (think of logs, metrics, or
+network packet data), and to send it to Elasticsearch, either directly or via
+Logstash, so it can be visualized with Kibana.
 
-This repository contains libbeat and all the officially supported Beats, in the
-following folders:
+This repository contains
+[libbeat](https://github.com/elastic/beats/tree/master/libbeat), our Go
+framework for creating Beats, and all the officially supported Beats:
 
-Folder  | Description
+Beat  | Description
 --- | ---
-[libbeat](https://github.com/elastic/beats/tree/master/libbeat) | The Go framework for creating new Beats
-[Filebeat](https://github.com/elastic/beats/tree/master/filebeat) | Lightweight log forwarder to Logstash & Elasticsearch
-[Heartbeat](https://github.com/elastic/beats/tree/master/heartbeat) | Testing service availability using PING based on ICMP, TCP or higher level protocols
```

Comment:
It was recently added in #2760. I think I agree to leave it out, since I change this list to be a a list of Beats, not so much a list of folders.

@ruflin any objections?